### DISCUSSION TITLE
Horizontal Button...button options

### DISCUSSION THREAD

**Discussion by ghxstxch** [Clooos/Bubble-Card#952] [Feature requests]
Hi, I apologize if I did not rtfm right or am missing something, but if it is not a feature already, may I add a suggestion to add the same "options" normal buttons do, like action on hold, tap, etc? Would like to be able to toggle lights by long press

**Comment by mirisbowring** [comment]
I would like to have this feature too.
I would like to have this footer bar but would like to toggle some booleans instead of opening / closing popups!
